cvss3/score_test.go

```diff
@@ -78,8 +78,7 @@ func TestBaseScores(t *testing.T) {
 
 	for i, c := range cases {
 		t.Run(fmt.Sprintf("case %2d", i+1), func(t *testing.T) {
-			v := Vector{}
-			v.Parse(c.str)
+			v, _ := VectorFromString(c.str)
```

Comment:
So, you only handle error in tests? :) Either don't return it, or handle it here too.

Reply:
I didn't check the error here because it should fail in a different test if this error can't be parsed. I'll add the check

Reply:
Somehow, I didn't realize this file was also a unittest one :( My bad, pls ignore.